[2024-11-18T11:33:01.570+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T11:33:01.614+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_pipeline.transform_weather_data manual__2024-11-18T11:27:45.127912+00:00 [queued]>
[2024-11-18T11:33:01.629+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_pipeline.transform_weather_data manual__2024-11-18T11:27:45.127912+00:00 [queued]>
[2024-11-18T11:33:01.630+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 6
[2024-11-18T11:33:01.655+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): transform_weather_data> on 2024-11-18 11:27:45.127912+00:00
[2024-11-18T11:33:01.673+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1373) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-18T11:33:01.676+0000] {standard_task_runner.py:72} INFO - Started process 1374 to run task
[2024-11-18T11:33:01.673+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_etl_pipeline', 'transform_weather_data', 'manual__2024-11-18T11:27:45.127912+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl.py', '--cfg-path', '/tmp/tmpl_3foeu9']
[2024-11-18T11:33:01.677+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask transform_weather_data
[2024-11-18T11:33:01.783+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_etl_pipeline.transform_weather_data manual__2024-11-18T11:27:45.127912+00:00 [running]> on host 527ca48b98a5
[2024-11-18T11:33:02.006+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='naphat' AIRFLOW_CTX_DAG_ID='weather_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T11:27:45.127912+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-18T11:27:45.127912+00:00'
[2024-11-18T11:33:02.008+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T11:33:02.011+0000] {python.py:240} INFO - Done. Returned value was: {'Time of Record': '2024-11-18 11:23:46', 'Country': 'TH', 'City': 'Bangkok', 'Latitude': 13.75, 'Longitude': 100.5167, 'Current Weather': 'Clouds', 'Temperature (F)': 85.892, 'Feels Like (F)': 89.54600000000005, 'Minimun Temp (F)': 84.90200000000007, 'Maximum Temp (F)': 85.892, 'Pressure': 1008, 'Humidty': 56, 'Wind Speed': 1.9, 'Sunrise (Local Time)': '2024-11-17 23:19:15', 'Sunset (Local Time)': '2024-11-18 10:47:17'}
[2024-11-18T11:33:02.056+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T11:33:02.057+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_etl_pipeline, task_id=transform_weather_data, run_id=manual__2024-11-18T11:27:45.127912+00:00, execution_date=20241118T112745, start_date=20241118T113301, end_date=20241118T113302
[2024-11-18T11:33:02.117+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T11:33:02.148+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-18T11:33:02.151+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
